### PR TITLE
add 17 abbreviation to db

### PR DESCRIPTION
**Pull request by COL-js**
## Changes proposed

I added 17 new abbreviations which are CMIIW, B2W, AFK, CYDM, OFC, AMA, AWOL, B4N, BAK, BAU, BITMT, DILLIGAF, F2P, F2F, FFA, GWHTLC and ICAM.

## Check List (Check all the applicable boxes)

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Note to reviewers
This is my first time participating in open source. if there is any problem with my code, i am sorry.

### CODE REVIEW

File: public/server/db.json [hunk 1]
```diff
@@ -109,6 +109,57 @@
     },
     "ootd":{
         "definition": "Outfit Of The Day"
-    }
-    
+    },
+    "cmiiw":{
+        "definition": "Correct me if i'm wrong"
```
Comment (Njong392) [inline public/server/db.json:114]: These all look AMAZING! Each letter of each word in the definition has to be uppercase though. Please look over your additions again. 
```suggestion
        "definition": "Correct Me If I'm Wrong"
```

File: public/server/db.json [hunk 2]
```diff
@@ -109,6 +109,57 @@
...
+        "definition": "Check your Direct Message"
+    },
+    "ofc":{
+        "definition": "Of course"
+    },
+    "ama":{
+        "definition": "Ask Me Anything"
+    },
+    "awol":{
+        "definition": "Absence without Leave",
```
Comment (Njong392) [inline public/server/db.json:132]: I think the main definition is 'Absent Without Official leave'?
```suggestion
        "definition": "Absent Without Official Leave",
```

File: public/server/db.json [hunk 3]
```diff
@@ -109,6 +109,57 @@
...
+        "alternatives": "Away without Leave"
```
Comment (Njong392) [inline public/server/db.json:133]: Alternatives could now be, 
```suggestion
        "alternatives": "Away Without Leave, Absent Without Leave"
```

File: public/server/db.json [hunk 4]
```diff
@@ -109,6 +109,57 @@
...
+        "definition": "Back to Keyboard"
+    },
+    "bau":{
+        "definition": "Business As Usual"
+    },
+    "bitmt":{
+        "definition": "But in the maintime"
+    },
+    "dilligaf":{
+        "definition": "Did i look like i give a fuck"
```
Comment (Njong392) [inline public/server/db.json:148]:
```suggestion
        "definition": "Do I Look Like I Give A Fuck"
```

File: public/server/db.json [hunk 5]
```diff
@@ -109,6 +109,57 @@
...
+        "definition": "Face to Face"
+    },
+    "ffa":{
+        "definition": "Free for All"
+    },
+    "gwhtlc":{
+        "definition": "Glad we had this little chat"
+    },
+    "icam":{
+        "definition": "I can't agree more"
```
Comment (Njong392) [inline public/server/db.json:163]:
```suggestion
        "definition": "I couldn't agree more"
```

**Review by Njong392 — CHANGES_REQUESTED** [review]
These look great :100: Thank you the HUGE contribution. Do look at the changes I've suggested.

**Comment by Njong392** [comment]
I suggest you hold on for a bit with this pull request for the time being. We are changing the structure of the JSON format, and I'll review these once we are done :+1:

**Comment by COL-js** [comment]
Okay

File: public/server/db.json [hunk 6]
```diff
@@ -109,6 +109,57 @@
...
     "ootd":{
         "definition": "Outfit Of The Day"
-    }
-    
+    },
+    "cmiiw":{
+        "definition": "Correct me if i'm wrong"
+    },
+    "b2w":{
+        "definition": "Back to work"
```
Comment (Njong392) [inline public/server/db.json:117]: The first letter of each word should be in capital letters. Do this for every new abbreviation you added.
```suggestion
        "definition": "Back To Work"
```

Comment (COL-js) [reply public/server/db.json:117]: Sorry about that, I thought I worked on it all. I will do that now

**Comment by COL-js** [comment]
I've edited it, you can check it out

**Pull request merged**